src/network/invthread.py

```diff
@@ -40,10 +40,10 @@ class InvThread(StoppableThread):
     @staticmethod
     def handleLocallyGenerated(stream, hashId):
         """Locally generated inventory items require special handling"""
-        dandelion.instance.addHash(hashId, stream=stream)
+        dandelion_enbled = dandelion_ins.addHash(hashId, stream=stream)
```

Comment:
As per https://github.com/Bitmessage/PyBitmessage/pull/2246#discussion_r1618334222. i have returned dandelion enabled value here as we. do i need to use `dandelion_ins.enabled` here as well?

Reply:
I think using `dandelion_ins.enabled` makes more sense.

Reply:
Done